src/composable/orderTypes/Twap.ts

```diff
@@ -350,6 +351,14 @@ export class Twap extends ConditionalOrder<TwapData, TwapStruct> {
     return undefined
   }
 
+  protected async polledOrderInOrderbook(
+    _orderUid: string,
+    _order: GPv2Order.DataStructOutput,
+    _params: PollParams
+  ): Promise<PollResultErrors | undefined> {
+    return undefined
```

Comment:
Is it not used at all on TWAP?

Reply:
Not for now... wait for one of the next PRs (see the not included section)